src/styles/index.scss

```diff
@@ -502,7 +502,8 @@ input[type='email']:focus {
 			color: white;
 		}
 		p {
-			@include para(1.1rem, 1.3rem);
+			@include para(1rem, 1.8rem);
```

Comment:
```suggestion
			@include para(1rem, 1.6rem);
```

Reply:
Adjusting the text spacing in the y plane helps with visual alignment between components. We have some work to do with margins and padding between components, but getting text spacing consistent between components in a PR will reduce our technical debt going forward. See attached images for examples.

Mission statement spacing:
![acm_mission](https://user-images.githubusercontent.com/29528527/67646533-ce150d80-f8eb-11e9-8e27-fcb515bba154.png)

PR changes:
![acm_socmed_spacing](https://user-images.githubusercontent.com/29528527/67646557-f69d0780-f8eb-11e9-9e7f-a2fd8134d67f.png)

Suggested change:
![acm_socmed_new](https://user-images.githubusercontent.com/29528527/67646573-0b799b00-f8ec-11e9-9948-14e61ce98256.png)